moto/sqs/responses.py

```diff
@@ -282,7 +284,7 @@ def receive_message(self):
         <VisibilityTimeout>{{ queue.visibility_timeout }}</VisibilityTimeout>
     </CreateQueueResult>
     <ResponseMetadata>
-        <RequestId>7a62c49f-347e-4fc4-9331-6e8e7a96aa73</RequestId>
+        <RequestId>{{ requestid }}</RequestId>
```

Comment:
Sneaky! I actually love this. Users won't have to pass this in but it's clearly parameterized.

Reply:
Yeah that one was a test to see if boto would accept it, and it does. Will spam the request id's on all the sqs methods to see if it likes it